content/hi/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,407 @@
+
+```shell
+cat <<EOF | sudo tee /etc/modules-load.d/containerd.conf
+overlay
+br_netfilter
+EOF
+
+sudo modprobe overlay
+sudo modprobe br_netfilter
+
+# Setup required sysctl params, these persist across reboots.
+cat <<EOF | sudo tee /etc/sysctl.d/99-kubernetes-cri.conf
+net.bridge.bridge-nf-call-iptables  = 1
+net.ipv4.ip_forward                 = 1
+net.bridge.bridge-nf-call-ip6tables = 1
+EOF
+
+# Apply sysctl params without reboot
+sudo sysctl --system
+```
+
+कंटेनर स्थापित करें:
+
+{{< tabs name="tab-cri-containerd-installation" >}}
+{{% tab name="Linux" %}}
+
+1. आधिकारिक डॉकर रिपॉजिटरी से `containerd.io` पैकेज स्थापित करें।  
+अपने संबंधित लिनक्स वितरण के लिए डॉकर रिपॉजिटरी स्थापित करने और `containerd.io` पैकेज स्थापित करने के निर्देश यहां देखे जा सकते हैं
+[Install Docker Engine](https://docs.docker.com/engine/install/#server).
+
+2. कंटेनर को कॉन्फ़िगर करें:
+
+   ```shell
+   sudo mkdir -p /etc/containerd
+   containerd config default | sudo tee /etc/containerd/config.toml
+   ```
+
+3. कंटेनर को पुनरारंभ करें:
+
+   ```shell
+   sudo systemctl restart containerd
+   ```
+
+{{% /tab %}}
+{{% tab name="Windows (PowerShell)" %}}
+
+Powershell सत्र प्रारंभ करें, `$Version` को वांछित संस्करण पर सेट करें (उदाहरण: `$Version=1.4.3`), और फिर निम्न आदेश चलाएँ: 
+
+1. कंटेनरड डाउनलोड करें: 
+
+   ```powershell
+   curl.exe -L https://github.com/containerd/containerd/releases/download/v$Version/containerd-$Version-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
+   tar.exe xvf .\containerd-windows-amd64.tar.gz
+   ```
+
+2. एक्सट्रेक्ट एंड कॉन्फ़िगर:
+
+   ```powershell
+   Copy-Item -Path ".\bin\" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force
+   cd $Env:ProgramFiles\containerd\
+   .\containerd.exe config default | Out-File config.toml -Encoding ascii
+
+   # Review the configuration. Depending on setup you may want to adjust:
+   # - the sandbox_image (Kubernetes pause image)
+   # - cni bin_dir and conf_dir locations
+   Get-Content config.toml
+
+   # (Optional - but highly recommended) Exclude containerd from Windows Defender Scans
+   Add-MpPreference -ExclusionProcess "$Env:ProgramFiles\containerd\containerd.exe"
+   ```
+
+3. कंटेनर शुरू करें:
+
+   ```powershell
+   .\containerd.exe --register-service
+   Start-Service containerd
+   ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### `systemd` cgroup ड्राइवर {#containerd-systemd} का उपयोग करें
+
+`systemd` cgroup ड्राइवर {#containerd-systemd} का उपयोग करना
+`runc` के साथ `/etc/containerd/config.toml` में `systemd` cgroup ड्राइवर का उपयोग करने के लिए, सेट करें
+
+```
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+  ...
+  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+    SystemdCgroup = true
+```
+
+यदि आप इस परिवर्तन को लागू करते हैं, तो सुनिश्चित करें कि आप फिर से कंटेन्डर को पुनरारंभ करें: 
+
+```shell
+sudo systemctl restart containerd
+```
+
+Kubeadm का उपयोग करते समय, मैन्युअल रूप से कॉन्फ़िगर करें
```

Comment:
nit: If you can, reword this to avoid starting the sentence with “kubeadm”.

Reply:
जब भी आप `Kubeadm` का उपयोग करें, मैन्युअल रूप से कॉन्फ़िगर करें  @Darshnadas May be this can help